src/app/auth/commandAuth.ts

```diff
@@ -139,13 +153,44 @@ const submit = async (request: SubmitRequest): Promise<void> => {
         throw new AuthorizationError(
             `Thank you for giving bounty commands a try!\n` +
             `It looks like you don't have permission to ${request.activity} this bounty.\n` +
-            `This bounty has already been claimed by <@${dbBountyResult.claimedBy.discordId}>  (${dbBountyResult.claimedBy.discordHandle}). ` +
+            `This bounty has already been claimed by <@${dbBountyResult.claimedBy.discordId}>. ` +
```

Comment:
TODO: given the unknown or invalid user bug, either investigate, or move to a static name. 

mentionable a better UX since they can click on it and message right away, but for now, keep both